pmd-cpp/etc/grammar/cpp.jj

```diff
@@ -312,9 +312,69 @@ TOKEN :
 
   <  CHARACTER : ("L")? "'" ( ( ~["'","\\","\r","\n"] ) | ( "\\" ( ~["\n","\r"] ) ) )* "'" >
 
-| <  STRING : ("L")? "\"" ( ( ~["\"","\\","\r","\n"] ) | ( "\\" ( ~["\n","\r"] | "\n" | "\r\n" ) ) )* "\"" >
+| < #STRPREF : (["L", "u", "U"] | "u8")? >
+| <  STRING : <STRPREF>
+      "\"" ( ( ~["\"","\\","\r","\n"] ) | ( "\\" ( ~["\n","\r"] | "\n" | "\r\n" ) ) )* "\"" >
 
-| <  RSTRING : "R\"(" ( ~[")"] | ( ")" ~["\""] ) )* ")\"" >
+}
+
+// Raw C++11 string literal support
+// https://en.cppreference.com/w/cpp/language/string_literal
+TOKEN :
+{
+  <  RSTRING : <STRPREF> "R\"" >
+  {
+    StringBuffer sb = new StringBuffer(16);
```

Comment:
Is there a reason, why you used StringBuffer? I think, StringBuilder is enough

```suggestion
    StringBuilder sb = new StringBuilder(16);
```

Reply:
Good suggestion, I'll change this!